### ISSUE TITLE
Text inside backticks is escaped

### ISSUE THREAD

**Issue by ncalca** [xijo/reverse_markdown#46]
When I try to convert a `<p>` element containing text inside backticks, underscores (and asterisks)
are escaped:

```ruby
ReverseMarkdown.convert("<p>`foo_bar`</p>")
=> "`foo\\_bar`"
```

Then, when I try to convert back the result with `RedCarpet` then I get this:
```ruby
=> "<p><code>foo\\_bar</code></p>\n"
```
which is inconsistent with the reversed string due to the escaping.
Shouldn't reverse markdown recognise the backticks and skip escaping inside them?

**Comment by xijo** [comment]
@ncalca thanks for reporting. First impression: you're right! I'll take a closer look and see what the matter, alright?

**Comment by ncalca** [comment]
Thanks @xijo!

**Comment by xijo** [comment]
Fixes by @livathinos in #47. I'll release a new version in a couple of minutes.
